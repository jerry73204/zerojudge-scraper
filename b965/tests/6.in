9 7 5
9 4 8 5 2 2 1
0 8 2 0 1 5 6
5 7 1 7 4 0 0
7 9 3 3 0 5 5
2 4 2 3 0 5 3
5 8 2 4 6 9 8
9 7 5 4 0 7 1
0 0 9 5 0 9 1
3 6 6 8 3 9 7
0 0 0 0 1