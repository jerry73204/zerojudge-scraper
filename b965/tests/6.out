9 7
3 6 6 8 3 9 7
0 0 9 5 0 9 1
9 7 5 4 0 7 1
5 8 2 4 6 9 8
2 4 2 3 0 5 3
7 9 3 3 0 5 5
5 7 1 7 4 0 0
0 8 2 0 1 5 6
9 4 8 5 2 2 1